pom.xml

```diff
@@ -18,8 +24,44 @@
 
   <build>
     <finalName>${project.artifactId}</finalName>
+
+      <plugins>
+        <plugin>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-maven-plugin</artifactId>
+        </plugin>
+      </plugins>
   </build>
   <dependencies>
+
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-jpa</artifactId>
+      <version>3.3.0</version>
```

Comment:
You can remove <version>3.3.0</version> everywhere since you use 3.2.4 in spring-boot-starter-parent

Reply:
removed